FEM/rf_bc_new.cpp

```diff
@@ -1378,7 +1378,12 @@ void CBoundaryConditionsGroup::Set(CRFProcess* pcs, int ShiftInNodeVector, const
 					//					debug_out.close();
 					//#endif
 					std::vector<size_t> msh_nod_vec;
+					double computed_search_length = m_msh->getSearchLength();
+					if (bc->epsilon != -1) 
```

Comment:
It is better to use epsilon from gli. We can take a look together once you are back.

Reply:
no i have no further comments. i will merge this